### DISCUSSION TITLE
Battleground and Debuffs.

### DISCUSSION THREAD

**Discussion by HasieEST** [chromiecraft/chromiecraft#3310] [Suggestions]
Would be nice, if there was a logic, which would clear all player applied debuffs on player once they are zoned out of Battleground. This would result in players not accidentally getting nuked with dots to death right after the end of battleground.

**Comment by 55Honey** [comment]
Hi @HasieEST,
Is this a blizzlike mechanic? I think the current behaviour is how it was in retail.

**Reply by HasieEST** [reply]
Apparently not after digging around. It would be QoL change that isn't Blizz-like, but what happens next isn't my decision therefore I would eject myself from this discussion.
